include/cpp_lmdb/details/details.hpp

```diff
@@ -71,7 +71,7 @@ constexpr auto make_tx(
         return std::unexpected{result};
     }
 
-    return txn_unique_ptr_t{txn, details::txn_deleter<LmdbApi>{api}};
+    return txn_unique_ptr_t<LmdbApi>{txn, details::txn_deleter<LmdbApi>{api}};
```

Comment:
Sadly, this C++20 feature seems to be not supported by clang
![image](https://github.com/vvish/lmdb-cpp/assets/14174805/b6e8e7fc-a82e-49cd-a4fe-338d1f004977)
Thank you for adopting the code.